TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift

```diff
@@ -30,6 +30,21 @@ public typealias FactorListSuccessBlock = ([Factor]) -> ()
 ///:nodoc:
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/5d0f0dc7fcbbb876a57dfd6fbf71f6a800f5ce49/TwilioVerifySDK/TwilioVerify/Sources/Domain/Manager/TwilioVerify.swift#L30"></span>Prefer at least one space after slashes for comments.<br />
<code>comment_spacing</code> <code>TwilioVerify.swift:30</code></td>
    </tr>
  </tbody>
</table>